docs/index.md

```diff
@@ -32,11 +32,10 @@ Its rules look like the code you already write; no abstract syntax trees, regex
 
 The Semgrep ecosystem includes:
 
-* [Semgrep](getting-started/) - the open-source command line tool at the heart of everything
-* [Semgrep CI](semgrep-ci/overview/) - a specialized Docker image for running Semgrep in CI environments
-* [Semgrep Playground](https://semgrep.dev/editor) - an online interactive tool for writing and sharing rules
-* [Semgrep Registry](https://semgrep.dev/explore) - 2,000+ community-driven rules covering security, correctness, and performance bugs
-* [Semgrep App](https://semgrep.dev/manage) - deploy, manage, and monitor Semgrep at scale with free and paid tiers
+* [Semgrep](getting-started/) - The open-source command line tool at the heart of everything. You can run it also in [CI environment](semgrep-ci/overview/)!
```

Comment:
```suggestion
* [Semgrep](getting-started/) - The open-source command line tool at the heart of everything.
```

(I propose moving the CI mention to the App bullet point, see the suggestion below)